Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader

```diff
@@ -111,12 +132,12 @@ Shader "Mixed Reality Toolkit/Wireframe"
 
                 // Fade out the alpha but not the color so we don't get any weird halo effects from
                 // a fade to a different color.
-                float4 color = I * _WireColor + (1 - I) * _BaseColor;
-                color.a = I;
-                return color;
+                return I * _WireColor + (1 - I) * _BaseColor;
```

Comment:
The comment still mentions doing something special with the alpha vs the color, but it looks like the new code doesn't explicitly do anything with `.a` anymore? Will this now cause the "weird halo effects"?

Reply:
Ah yes, I need to remove that comment. I don't believe that comment was valid to begin with and was written in error. I have not experienced any halo effects with my changes.